src/backend/dd/DDSimDebug.cpp

```
@@ -38,6 +38,7 @@
 
 #include <Eigen/Dense>
 #include <algorithm>
+#include <cctype>
```

Cppcheck (2.18.0)

[information] Include file

(missingIncludeSystem)